pkg/iac/terraform/state/test/aws_default_route_table/terraform.tfstate

```diff
@@ -0,0 +1,108 @@
+{
+  "version": 4,
+  "terraform_version": "1.2.2",
+  "serial": 4,
+  "lineage": "14d1f567-0bef-b33c-81ac-3c2a27a9d155",
+  "outputs": {},
+  "resources": [
+    {
+      "mode": "managed",
+      "type": "aws_default_route_table",
+      "name": "default",
+      "provider": "provider[\"registry.terraform.io/hashicorp/aws\"]",
+      "instances": [
+        {
+          "schema_version": 0,
+          "attributes": {
+            "arn": "arn:aws:ec2:us-east-1:282061376516:route-table/rtb-03fda555c6c35a1e5",
+            "default_route_table_id": "rtb-03fda555c6c35a1e5",
+            "id": "rtb-03fda555c6c35a1e5",
+            "owner_id": "282061376516",
+            "propagating_vgws": null,
+            "route": [],
+            "tags": null,
+            "tags_all": {},
+            "timeouts": {
+              "create": "6m",
+              "update": "3m"
+            },
+            "vpc_id": "vpc-092fed41ffac6df72"
+          },
+          "sensitive_attributes": [],
+          "private": "eyJlMmJmYjczMC1lY2FhLTExZTYtOGY4OC0zNDM2M2JjN2M0YzAiOnsiY3JlYXRlIjozNjAwMDAwMDAwMDAsInVwZGF0ZSI6MTgwMDAwMDAwMDAwfX0=",
+          "dependencies": [
+            "aws_default_vpc.default"
+          ]
+        }
+      ]
+    },
+    {
+      "mode": "managed",
+      "type": "aws_default_vpc",
```

Comment:
You can remove everything but aws_route_table resource from that state file. It will be an inconsistent one regarding Terraform logic but we do not care here it's only to ensure unmarshalling is going well !
We try to minimise test maintenance by removing side effects 😉 

You can also remove the lock file from that folder, it is not used.

You'll also have to update the result file.

Reply:
> You can also remove the lock file from that folder, it is not used.

☝🏻 `pkg/iac/terraform/state/test/aws_default_route_table/.terraform.lock.hcl` need to be removed too